spec/presenters/search_result_presenter_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe SearchResultPresenter do
+  let(:presenter) { described_class.new(model) }
+
+  describe "#entity_display_name" do
+    let(:model) { Object.new }
+
+    it { expect(presenter).to respond_to(:entity_display_name) }
+  end
+
+  describe "#is_a? WasteCarriersEngine::NewRegistration" do
```

Comment:
What does the # at the beginning mean? Is it to do with testing a method? :)

Reply:
It's an rspec convention. For an instance method we could write `describe "#instance_method_name"` and for a class method `describe ".class_method_name"`